frontend/src/component/admin/users/UsersList/ResetPassword/ResetPassword.tsx

```diff
@@ -0,0 +1,124 @@
+import React, { useState } from 'react';
+import classnames from 'classnames';
+import { Box, styled, TextField, Typography } from '@mui/material';
+import { modalStyles } from 'component/admin/users/util';
+import { Dialogue } from 'component/common/Dialogue/Dialogue';
+import { useThemeStyles } from 'themes/themeStyles';
+import { IUser } from 'interfaces/user';
+import useAdminUsersApi from 'hooks/api/actions/useAdminUsersApi/useAdminUsersApi';
+import { UserAvatar } from 'component/common/UserAvatar/UserAvatar';
+import useToast from 'hooks/useToast';
+import { formatUnknownError } from 'utils/formatUnknownError';
+import { LinkField } from '../../LinkField/LinkField';
+
+const StyledUserAvatar = styled(UserAvatar)(({ theme }) => ({
+    width: theme.spacing(5),
+    height: theme.spacing(5),
+    margin: 0,
+}));
+
+interface IChangePasswordProps {
+    showDialog: boolean;
+    closeDialog: () => void;
+    user: IUser;
+}
+
+const ResetPassword = ({
+    showDialog,
+    closeDialog,
+    user,
+}: IChangePasswordProps) => {
+    const { classes: themeStyles } = useThemeStyles();
+    const { resetPassword } = useAdminUsersApi();
+    const { setToastApiError } = useToast();
+    const [resetLink, setResetLink] = useState('');
+
+    const submit = async (event: React.SyntheticEvent) => {
+        event.preventDefault();
+        if (!user.email) {
+            setToastApiError('User without email cannot reset password');
+            return;
+        }
+
+        try {
+            const token = await resetPassword(user.email).then((res) =>
+                res.ok ? res.json() : undefined,
+            );
+
+            if (token) {
+                console.log(token);
+                setResetLink(token.resetPasswordUrl);
+            } else {
+                setToastApiError("Could not reset password");
+            }
+        } catch (error) {
+            console.warn(error);
```

Comment:
I think setting the toast is enough here

Reply:
Done: https://github.com/Unleash/unleash/pull/5580/commits/2b555473f994f5e4c05f919d1a94a96f069a3298